gear/gtest/spec/test_nft_example.yaml

```diff
@@ -0,0 +1,572 @@
+title: Test NFT-example
+
+programs:
+  - id: 1
+    path: nft_example.wasm
+    init_message:
+      kind: custom
+      value:
+        name: "NFT collection"
+        symbol: "NFT"
+        base_uri: "https://"
+
+fixtures:
+  - title: mint/burn
+    messages:
+      # Mints the first token, expects successfull reply
+      - &mint
+        destination: 1
+        source: &source
+          kind: id
+          value: 2
+        payload: &payload_mint
+          kind: custom
+          value:
+            mint: 
+      # Mints the second token, expects successfull reply
+      - *mint
+      # Burns the second token, expects successfull reply
+      - &burn
+        destination: 1
+        source: *source
+        payload:
+          kind: custom
+          value:
+            burn: 1
+      # Retries to burn the second token, must fail
+      - *burn
+      # Not owner tries to burn the first token, must fail
+      - &burn
+        destination: 1
+        source:
+          kind: id
+          value: 3
+        payload:
+          kind: custom
+          value:
+            burn: 0
+
+    expected:
+      - allowError: true
+        log:
+          # successfull reply of mint
+          - &mint_success
+            destination: 2
+            payload:
+              kind: custom
+              value:
+                transfer:
+                  from: &zero 0x0000000000000000000000000000000000000000000000000000000000000000
+                  to: &user_1 0x0200000000000000000000000000000000000000000000000000000000000000
+                  token_id: 0 
+          # successfull reply of mint              
+          - <<: *mint_success
```

Comment:
I don't understand why `<<: *mint_success` is used here, is intention is to change value of `token_id: 1` in previous definition of `mint_success`?

Reply:
Yes

Reply:
but then it almost writing whole `mint_success` but I hope it is being used later.

Reply:
I just wanted to indicate that it's the same type of message (about successful mint) :)